diff --git a/YUView.pro b/YUView.pro
index 2ed8e70d..548b33bf 100644
--- a/YUView.pro
+++ b/YUView.pro
@@ -1,9 +1,7 @@
 TEMPLATE = subdirs
-SUBDIRS = YUViewLib YUViewApp YUViewUnitTest
+SUBDIRS = YUViewLib YUViewApp
 
 YUViewApp.subdir = YUViewApp
 YUViewLib.subdir = YUViewLib
-YUViewUnitTest.subdir = YUViewUnitTest
 
 YUViewApp.depends = YUViewLib
-YUViewUnitTest.depends = YUViewLib
diff --git a/YUViewApp/YUViewApp.pro b/YUViewApp/YUViewApp.pro
index 2f1d5742..05b41ce9 100644
